pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -147,20 +148,31 @@ func (mi *StorageInitializerInjector) InjectStorageInitializer(pod *v1.Pod) erro
 		ReadOnly:  true,
 	}
 	userContainer.VolumeMounts = append(userContainer.VolumeMounts, sharedVolumeReadMount)
-
 	// Change the CustomSpecStorageUri env variable value to the default model path if present
 	for index, envVar := range userContainer.Env {
 		if envVar.Name == constants.CustomSpecStorageUriEnvVarKey && envVar.Value != "" {
 			userContainer.Env[index].Value = constants.DefaultModelLocalMountPath
 		}
 	}
+	podNamespace := pod.Namespace
```

Comment:
Should it fail if not found?

Reply:
I remember that pod.Namespace here will always be ""

Reply:
@hougangliu do you know why this is empty or

Reply:
I didn't investigate it further and just get namespace by `req.AdmissionRequest.Namespace`